From 87456bcef88083443e9c968fe209a46e8500de80 Mon Sep 17 00:00:00 2001
From: Callum Farmer <callumjfarmer13@gmail.com>
Date: Fri, 28 Aug 2020 17:25:29 +0100
Subject: [PATCH] Patch4 (#14)

* Patch3 (#11)

* Patch2 (#8)

* Patch1 (#3)

* Update configure.ac

* Update configure.ac

* Patch1 (#7)

* Update configure.ac

* Update configure.ac

* Update configure.ac

* Update configure.ac

* Update configure.ac
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index c573924..942457e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5,6 +5,8 @@ AC_INIT([budgie-screensaver],
         [4.0],
         [https://github.com/getsolus/budgie-screensaver])
 
+AC_CONFIG_MACRO_DIR([m4])
+AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_SRCDIR(src/gnome-screensaver.c)
 
 AM_INIT_AUTOMAKE([1.10 no-dist-gzip dist-xz tar-ustar])
@@ -33,7 +34,8 @@ GETTEXT_PACKAGE=budgie-screensaver
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [Name of default gettext domain])
 
-AM_GLIB_GNU_GETTEXT
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION([0.17])
 GLIB_GSETTINGS
 
 # Dependencies
diff --git a/src/Makefile.am b/src/Makefile.am
index 4d594d7..6ac785b 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -5,7 +5,7 @@ NULL =
 
 saverdir = $(libexecdir)/budgie-screensaver
 
-INCLUDES =							\
+AM_CPPFLAGS =							\
 	-I.							\
 	-I$(srcdir)						\
 	$(BUDGIE_SCREENSAVER_CFLAGS)				\
-- 

diff --git a/Makefile.am b/Makefile.am
index 4aa0deb..f7cb628 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -53,3 +53,4 @@ MAINTAINERCLEANFILES =		\
 
 
 -include $(top_srcdir)/git.mk
+ACLOCAL_AMFLAGS = -I m4
-- 
